src/frame/js/actions/conversation.js

```diff
@@ -88,33 +90,36 @@ const throttlePerUser = (userId) => {
 
 function postSendMessage(message) {
     return (dispatch, getState) => {
-        return core(getState()).appUsers.sendMessage(getUserId(getState()), message);
+        const {user: {_id}} = getState();
+        return dispatch(http('POST', `/appusers/${_id}/messages`, {
+            ...message,
+            role: 'appUser',
+            deviceId: getDeviceId()
```

Comment:
Obviously this wasn't decided when the PR was written, but this will change into `sender.client.id`. Can be done in a separate PR of course